lib/much-rails.rb

```diff
@@ -5,6 +5,7 @@
 
 require "much-rails/version"
 require "much-rails/action"
+require "much-rails/action/change_action"
```

Comment:
I wouldn't require this here. Requiring in "much-rails/action" should do it for you. So please move this to `much-rails/action`.